### PR TITLE
verify private elasticsearch endpoints

### PR DESCRIPTION
**Pull request by codyseibert**

### CODE REVIEW

File: web-api/verify-private-elasticsearch.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+ENV=$1
+
+endpoint=$(aws es describe-elasticsearch-domain --domain "efcms-search-$ENV" | jq -r ".DomainStatus.Endpoint" )
+
+echo $endpoint
+response=$(curl -I "https://$endpoint" | head -n 1 | cut -d$' ' -f2)
+
+if [[ "$response" != "403" ]]; then
+  echo "expected elasticsearch endpoint to thbe private"
```
Comment (rachaelparris) [inline web-api/verify-private-elasticsearch.sh]: Typo `thbe`

File: web-api/verify-private-elasticsearch.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+# Usage
+#   smoketest to verify the private buckets are configured as private
+
+# Arguments
+#   - $1 - the environment [dev, stg, prod, exp1, exp1, etc]
+
+[ -z "$1" ] && echo "The env to run smoketest to \$1 argument.  An example value of this includes [dev, stg, prod... ]" && exit 1
```
Comment (rachaelparris) [inline web-api/verify-private-elasticsearch.sh]: Should this be an error message? I think it might need to be updated to be more clear that it's an error.

Comment (rachaelparris) [reply web-api/verify-private-elasticsearch.sh]: Are AWS credentials required to run this? If so, could you add error messages if those environment variables are missing too?

**Review by rachaelparris — APPROVED** [review]

File: web-api/verify-private-s3-buckets.sh [hunk 3]
```diff
@@ -20,9 +20,11 @@ BUCKETS=(
 for bucket in ${BUCKETS[@]}; do
   # The only way to grant a bucket as public is via a bucket policy, so we check if the policy
   # file exists and throw an error on status codes
-  if aws s3api get-bucket-policy --bucket $bucket; then
+  $(aws s3api get-bucket-policy --bucket $bucket)
+  code=$?
+  if [ "${code}" == "0" ]; then
     echo "ustc-case-mgmt.flexion.us-documents-$ENV-us-east-1 is not private; it has a bucket policy when it should not"
```
Comment (kkoskelin) [inline web-api/verify-private-s3-buckets.sh]: `echo "$bucket is not private...` ?

File: web-api/verify-private-elasticsearch.sh (hunk 1 above)
Comment (kkoskelin) [inline web-api/verify-private-elasticsearch.sh]: `echo "expected elasticsearch $endpoint to be private"`? just for more detailed logging?

**Pull request merged**